### PR TITLE
theme: add thecyberden

### PR DESCRIPTION
**Pull request by jvsca**
### Prerequisites

- [x] I have read and understood the `CONTRIBUTING` guide
- [x] The commit message follows the [conventional commits][cc] guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added / updated (for bug fixes/features)

### Description

[Description of the change]
Added **thecyberden** theme.

[cc]: https://www.conventionalcommits.org/en/v1.0.0/#summary

### CODE REVIEW

File: themes/thecyberden.omp.json [hunk 1]
```diff
@@ -0,0 +1,128 @@
+{
+    "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/schema.json",
+    "final_space": false,
+    "console_title": false,
+    "console_title_style": "folder",
+    "blocks": [
+        {
+            "type": "prompt",
+            "alignment": "left",
+            "horizontal_offset": 0,
```
Comment (JanDeDobbeleer) [inline themes/thecyberden.omp.json]: these are default values and can be removed

File: themes/thecyberden.omp.json [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+                    }
+                },
+                {
+                    "type": "path",
+                    "style": "powerline",
+                    "powerline_symbol": "\uE0B0",
+                    "invert_powerline": false,
+                    "foreground": "#ffffff",
+                    "background": "#0080ff",
+                    "leading_diamond": "",
```
Comment (JanDeDobbeleer) [inline themes/thecyberden.omp.json]: as this segment is `powerline`, diamond settings should be removed

File: themes/thecyberden.omp.json [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+                    }
+                },
+                {
+                    "type": "git",
+                    "style": "powerline",
+                    "powerline_symbol": "\uE0B0",
+                    "invert_powerline": false,
+                    "foreground": "#193549",
+                    "background": "#c19c00",
+                    "leading_diamond": "",
```
Comment (JanDeDobbeleer) [inline themes/thecyberden.omp.json]: as this segment is `powerline`, diamond settings should be removed

File: themes/thecyberden.omp.json [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+                    "trailing_diamond": "",
+                    "properties": {
+                        "display_status": true,
```
Comment (JanDeDobbeleer) [inline themes/thecyberden.omp.json]: needs to be renamed to `fetch_status`, please validate the other properties as well (and add a `template`)

File: themes/thecyberden.omp.json [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+                    "style": "diamond",
+                    "powerline_symbol": "\uE0B0",
+                    "invert_powerline": false,
+                    "foreground": "#ffffff",
+                    "background": "#2e9599",
+                    "leading_diamond": "",
+                    "trailing_diamond": "\uE0B4",
+                    "properties": {
+                        "always_enabled": true,
+                        "color_background": true,
```
Comment (JanDeDobbeleer) [inline themes/thecyberden.omp.json]: `color_background` is deprecated and should be changed to levering `background_templates`

File: themes/thecyberden.omp.json [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+                    "properties": {
+                        "display_status": true,
+                        "display_stash_count": true,
+                        "display_upstream_icon": true
+                    }
+                },
+                {
+                    "type": "exit",
+                    "style": "diamond",
+                    "powerline_symbol": "\uE0B0",
```
Comment (JanDeDobbeleer) [inline themes/thecyberden.omp.json]: as this segment is `diamond`, powerline settings should be removed

File: themes/thecyberden.omp.json [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+                    "foreground": "#ff0000",
+                    "properties": {
+                        "prefix": "",
+                        "root_icon": "\uE00A"
+                    }
+                },
+                {
+                    "type": "shell",
+                    "style": "powerline",
+                    "powerline_symbol": "",
```
Comment (JanDeDobbeleer) [inline themes/thecyberden.omp.json]: this should be `plain` due to the absence of the `powerline_symbol`

**Review by JanDeDobbeleer — CHANGES_REQUESTED** [review]

File: themes/thecyberden.omp.json (hunk 1 above)
Comment (jvsca) [reply themes/thecyberden.omp.json]: Removed:
            "horizontal_offset": 0,
            "vertical_offset": 0,

File: themes/thecyberden.omp.json (hunk 2 above)
Comment (jvsca) [reply themes/thecyberden.omp.json]: Removed:
                    "leading_diamond": "",
                    "trailing_diamond": "\uE0B0",

File: themes/thecyberden.omp.json (hunk 3 above)
Comment (jvsca) [reply themes/thecyberden.omp.json]: Removed:
                    "leading_diamond": "",
                    "trailing_diamond": "",

File: themes/thecyberden.omp.json (hunk 4 above)
Comment (jvsca) [reply themes/thecyberden.omp.json]: Renamed and added template:
                        "fetch_status": true,
                        "fetch_stash_count": true,
                        "fetch_upstream_icon": true,
                        "template": "{{ .UpstreamIcon }}{{ .HEAD }}{{ .BranchStatus }}{{ if .Working.Changed }} \uF044 {{ .Working.String }}{{ end }}{{ if and (.Working.Changed) (.Staging.Changed) }} |{{ end }}{{ if .Staging.Changed }} \uF046 {{ .Staging.String }}{{ end }}{{ if gt .StashCount 0 }} \uF692 {{ .StashCount }}{{ end }}"

File: themes/thecyberden.omp.json (hunk 5 above)
Comment (jvsca) [reply themes/thecyberden.omp.json]: Removed:
                        "color_background": true,
                        "error_color": "#f1184c",
Added:
                    "background_templates": [
                        "{{ if gt .Code 0 }}#f1184c{{ end }}"
                    ],

File: themes/thecyberden.omp.json (hunk 6 above)
Comment (jvsca) [reply themes/thecyberden.omp.json]: Removed:
                    "powerline_symbol": "\uE0B0",
                    "invert_powerline": false,

File: themes/thecyberden.omp.json (hunk 7 above)
Comment (jvsca) [reply themes/thecyberden.omp.json]: Changed:
                    "style": "plain",

**Pull request merged**